src/collector.ts

```diff
@@ -105,9 +105,38 @@ class NaiveGomodParser {
     dependencies: Array<IDependency>;
 
     static parseDependencies(contents:string, goImports: Set<string>): Array<IDependency> {
+        let replaceDependencies = new Map();
+
+        function replaceMap(line: string, index: number) {
+            // skip any text after '//'
+            if (line.includes("//")) {
+                line = line.split("//")[0];
+            }
+            // split the replace statements by '=>'
+            const parts: Array<string> = line.replace('replace', '').replace('(', '').replace(')', '').trim().split('=>');
+            const replaceWith: Array<string> = (parts[1] || '').trim().split(' ');
```

Comment:
This line can also be moved in 'if' condition, trim() and split() can be avoided for the lines which do not have version string.